apps/web/src/server/applications/case/project-team/applications-project-team.controller.js

```diff
@@ -18,18 +18,43 @@ import {
 /** @typedef {import("../../../app/auth/auth-session.service.js").SessionWithAuth} SessionWithAuth */
 
 export const allRoles = [
-	{ value: 'case_manager', text: 'Case Manager' },
-	{ value: 'environmental_services', text: 'Environmental Services' },
-	{ value: 'inspector', text: 'Inspector' },
-	{ value: 'lead_inspector', text: 'Lead Inspector' },
-	{ value: 'legal_officer', text: 'Legal Officer' },
-	{ value: 'NSIP_administration_officer', text: 'NSIP Administration Officer' },
-	{ value: 'NSIP_officer', text: 'NSIP Officer' },
-	{ value: 'officer', text: 'Officer' },
-	{ value: 'operations_lead', text: 'Operations Lead' },
-	{ value: 'operations_manager', text: 'Operations Manager' }
+	{ value: 'case_manager', text: 'Case Manager', unique: true },
+	{ value: 'environmental_services', text: 'Environmental Services', unique: true },
+	{ value: 'inspector', text: 'Inspector', unique: false },
+	{ value: 'lead_inspector', text: 'Lead Inspector', unique: true },
+	{ value: 'legal_officer', text: 'Legal Officer', unique: true },
+	{ value: 'NSIP_administration_officer', text: 'NSIP Administration Officer', unique: false },
+	{ value: 'NSIP_officer', text: 'NSIP Officer', unique: false },
+	{ value: 'officer', text: 'Officer', unique: false },
+	{ value: 'operations_lead', text: 'Operations Lead', unique: true },
+	{ value: 'operations_manager', text: 'Operations Manager', unique: true }
 ];
 
+/**
+ * Returns available roles.
+ *
+ * Available roles include:
+ * - The role already assigned to the team member
+ * - The roles that can be assigned to multiple team members
+ * - The unassigned roles that cannot be assigned to multiple team members
+ *
+ * @param {Partial<ProjectTeamMember>} projectTeamMember
+ * @param {Partial<ProjectTeamMember>[]} projectTeamMembers
+ */
+function availableRoles(projectTeamMember, projectTeamMembers) {
```

Comment:
this function and the `const` on the next line have the same name - i'd use different ones